scheme/cca-realm-provisioning/corim_extractor.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca_realm_provisioning
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/services/handler"
+)
+
+type CorimExtractor struct{}
+
+func (o CorimExtractor) RefValExtractor(
```

Comment:
Rename to `ExtractRefVal`.  In general, methods should be verbs (unless constructors, factories, properties, etc). The name as-is suggests that this should return an extractor, rather than a slice of ref vals.

(note: I realise that this is meant to be implementing `common.IExtractor`, however there is no need for that interface -- this is only used within this code, and is never passed via the interface.)

Reply:
IExtractor is the interface which is been implemented by CorimExtractor. To your comment: _however there is no need for that interface -- this is only used within this code_

is not true, `IExtractor `  interfaces are invoked from: UnsignedCorimDecoder.go
hence this is not a local function!